src/components/ProfileForm/DogForm/dogForm.js

```diff
@@ -1,68 +0,0 @@
-import React, { Component } from "react";
```

Comment:
are you deleting this file entirely?

Reply:
Yes, it was used as an intermedia component between user profile and dog profile, and now we only use userform.js to direct to different components in different steps